script.js

```diff
@@ -0,0 +1,432 @@
+$(document).ready(function () {
```

Comment:
There is really a lot of repetition in this file, you could have tried to DRY it out a little bit more

An object for your character would have really helped you out here.